src/libraries/System.Net.Http.Json/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
```

Comment:
Since this is a new library and it didn't exist in full framework before, I believe the `StrongNameKeyId` should be `Open` instead of `Microsoft`. cc: @ericstj

Reply:
Right.  Please use Open Key, you *should* be able to omit `StrongNameKeyId` to get that behavior.

Reply:
I see other projects still have `<StrongNameKeyId>Open</StrongNameKeyId>` -- I think it is good to have it there so that when reading D.B.props people know immediately what key it is using without having to understand defaults, ask or look around?

Reply:
I left it explicit, if you want me to remove it, please let me know.